README.md

```diff
@@ -10,11 +10,11 @@ This component is an updated version of the [original Tahoma integration](https:
 
 ### Manual
 
-Copy the `custom_components/tahoma` to your `custom_components` folder. Reboot home assistant and install the Tahoma integration via the integrations config flow.
+Copy the `custom_components/tahoma` to your `custom_components` folder. Reboot Home Assistant and install the Tahoma integration via the integrations page.
```

Comment:
I would say "configure the Tahoma integration" instead of "install"

Reply:
Can you give it some love in another PR? I guess there are more parts that benefit from more clear text, especially around how to install beta's and the risk.